Integrated-Windows-Auth-Flow/src/main/java/IntegratedWindowsAuthFlow.java

```diff
@@ -1,82 +1,74 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import com.microsoft.aad.msal4j.IAccount;
 import com.microsoft.aad.msal4j.IAuthenticationResult;
 import com.microsoft.aad.msal4j.IntegratedWindowsAuthenticationParameters;
+import com.microsoft.aad.msal4j.MsalException;
 import com.microsoft.aad.msal4j.PublicClientApplication;
+import com.microsoft.aad.msal4j.SilentParameters;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.net.HttpURLConnection;
-import java.net.URL;
 import java.util.Collections;
-import java.util.concurrent.Future;
+import java.util.Set;
 
 public class IntegratedWindowsAuthFlow {
 
-    private final static String PUBLIC_CLIENT_ID = "Enter_the_Application_Id_here";
-    private final static String USER_NAME = "Enter_User_Name";
-    private final static String AUTHORITY_ORGANIZATION = "https://login.microsoftonline.com/organizations/";
-    private final static String GRAPH_DEFAULT_SCOPE = "https://graph.microsoft.com/.default";
+    private final static String CLIENT_ID = "";
```

Comment:
In general Application configurations like this should be kept i separate properties file,
like it is done is other samples. 
hardcoding config is anti patern.

Reply:
IMO for such a simple sample like these, it's much more convenient to just have these values here. Less overhead for people who are just getting started.

Reply:
yes, but it is well recognized anti pattern, which should be avoided , samples should be prod  level quality